unison-src/transcripts/suffixes.output.md

```diff
@@ -13,31 +13,35 @@ optional.isNone = cases
   Some _ -> false
 ```
 
+This also affects commands like find. Notice lack of qualified names in output:
+
 ```ucm
+.> add
 
-  I found and typechecked these definitions in scratch.u. If you
-  do an `add` or `update`, here's how your codebase would
-  change:
+  ⍟ I've added these definitions:
   
-    ⍟ These new definitions are ok to `add`:
-    
-      foo.bar.a       : Int
-      optional.isNone : Optional a -> Boolean
-   
-  Now evaluating any watch expressions (lines starting with
-  `>`)... Ctrl+C cancels.
+    foo.bar.a       : Int
+    optional.isNone : Optional a -> Boolean
```

Comment:
Note to self: This is just the typechecking message being replaced with the `add` output.